forms-html/Ahmed+Ebraheem/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>The Form</title>
+    </head>
+<body>
+<form>
+<label for="email">Enter your Email:</label><br>
+    <input type="text" id="email" name="email" value="Ahmed@Ebraheem"><br>
+<label for="Username">Username:</label><br>
+    <input type="text" id="Username" value="osama"><br>
```

Comment:
after each input label pair out new line and make sure that your code format is correct so the code will be like this

```
    <label for="email">Enter your Email:</label><br>
    <input type="text" id="email" name="email" value="Ahmed@Ebraheem"><br>

    <label for="Username">Username:</label><br>
    <input type="text" id="Username" value="osama"><br>

    <label for="pass">Password:</label><br>
    <input type="password" id="pass" name="pass" value="213233"><br>

    <label for="quan">Quantity (between 1 and 5):</label>
    <input type="number" id="quan" name="quantity" min="1" max="5" value="3"><br>

    <label for="volume">Volume (between 0 to 50):</label>
    <input type="range" id="volume" name="volume" min="0" max="50" value="25"><br><br>